import mysql.connector
from mysql.connector import MySQLConnection, Error
import subprocess
db = mysql.connector.connect(host='127.0.0.1',
                             user='root',
                             passwd='xxx'
                             )
cursor = db.cursor()
if db.is_connected():
    db_Info = db.get_server_info()
    print("Connected to MySQL database... MySQL Server version on ", db_Info)




with open(r'C:\Users\xxxxx\Documents\mysql\import.txt', newline='') as f:
    for lines in f:
        fields = lines.rstrip().split(";")
        first = str(fields[0])
        second = str(fields[1])
        third = str(fields[2])
        fourth = str(fields[3])
        sql = "INSERT INTO `test`.`deneme` (`ID`, `FIRSTNAME`,`LASTNAME`,`EMAIL`) VALUES ('%s', '%s', '%s', '%s');" % (first, second, third, fourth)
        print(sql)
        try:
            cursor.execute(sql)
            db.commit()
        except:
            db.rollback()





